src/account/account.service.ts

```diff
@@ -64,7 +64,12 @@ export class AccountService {
         private readonly events: AsyncEvents,
         private readonly accountRepository: KnexAccountRepository,
         private readonly fedifyContextFactory: FedifyContextFactory,
-        private readonly generateKeyPair: () => Promise<CryptoKeyPair> = generateCryptoKeyPair,
+        private readonly generateKeyPair: () => Promise<{ publicKey: CryptoKey, privateKey: CryptoKey }[]> = async () => {
```

Comment:
This method name needs updating to reflect the new return type (singular -> plural).

Why have we stopped using the `CryptoKeyPair` type here?